testing/bdd/test_ebuttd_resolved_timings_on_elements.py

```diff
@@ -0,0 +1,84 @@
+from pytest_bdd import scenarios, when, then, given, parsers
+import xml.etree.ElementTree as ET
+
+scenarios('features/timing/ebuttd_resolved_timings_on_elements.feature')
+
+@then('p resulted begin time is <p_resulted_begin_time>')
+def then_it_has_p_computed_time(test_context, p_resulted_begin_time):
```

Comment:
Please make sure the function names match what the functions do - here and in other places they don't match.